webapp/tests/karma/unit/services/telemetry.js

```diff
@@ -159,6 +198,13 @@ describe('Telemetry service', () => {
         chai.expect(aggregatedDoc.metadata.year).to.equal(2018);
         chai.expect(aggregatedDoc.metadata.month).to.equal(9);
         chai.expect(aggregatedDoc.metadata.user).to.equal('greg');
+        console.log(aggregatedDoc.metadata.versions);
+        chai.expect(aggregatedDoc.metadata.versions).to.deep.equal({
```

Comment:
@garethbowen re:
> You've updated the unit testing so it provides all the additional info, but I couldn't see any new expectations for the new info. I

Is that not this? Or do you mean something else?

Reply:
Oh yeah, that! I missed it. Thanks.